src/transaction/log_recovery_analysis.cpp

```diff
@@ -2204,35 +2210,19 @@ corruption_checker::check_log_record (const log_lsa &record_lsa, const log_rec_h
     }
 }
 
-/* log_recovery_analysis_from_transaction_table_snapshot - perform recovery for a passive transaction server
- *                  starting from a [recent] transaction table snapshot relayed via log and page server from
- *                  the active transaction server
- *
- * most_recent_trantable_snapshot_lsa (in): the lsa where a record containing, as payload, the packed contents
- *                                of a recent transaction table snapshot; starting from that snapshot, analyze the log
- *                                to construct an actual starting transaction table
- */
-void
-log_recovery_analysis_from_transaction_table_snapshot (THREAD_ENTRY *thread_p,
-    log_lsa most_recent_trantable_snapshot_lsa)
+static int
```

Comment:
A comment header to describe the usage would be useful

Reply:
Done